### DISCUSSION TITLE
Fetch to local API endpoint does not fire

### DISCUSSION THREAD

**Discussion by aceaspades-worldspark** [vercel/next.js#45338] [Help] [answered]
I'm attempting to call an API endpoint located at 'src/pages/api/forwardRequest' via a fetch request being called from a useSWR fetcher (getAccount). useSWR is properly calling the fetcher periodically, but the fetch request itself is _not_ firing, or at least it's getting stuck in some sort of limbo. 

The endpoint itself isn't being hit at all, oddly.

All the relevant code here in CodeSandbox: https://codesandbox.io/p/sandbox/frosty-matan-5kveoh

**Comment by wwwdepot** [comment, marked as answer]
you need to return the `res.json()` in the `ApiService.ts`

```diff
  }).then((res) => {
    console.log("but this will not...");
+    return res.json();
  });
```

**Reply by aceaspades-worldspark** [reply]
Thanks for catching that. The problem persists, however; `console.log("but this will not...")` is not being engaged at all, nor is the fetch itself that this statement is chained onto.

**Reply by wwwdepot** [reply]
Remove the `body` from the fetcher. GET request cannot have body.

**Reply by aceaspades-worldspark** [reply]
> Remove the `body` from the fetcher. GET request cannot have body.

THAT did the trick! Thanks!
